nixjs-rt/src/builtins/builtins.ts

```diff
@@ -22,6 +24,25 @@ export function getBuiltins() {
       );
     },
 
+    import: (path) => {
+      const pathStrict = path.toStrict();
```

Comment:
Not related to your PR, but just wondering if we can find another mechanism for lazy evaluation. Having to call this `toStrict()` method is a bit cumbersome. Any ideas are welcome.

Reply:
Isn't this only invoked when an import is already being evaluated? Like an import invocation is only evaluated when the value of it is required, and when that value is required then the path also needs to be evaluated

Reply:
Oh wait I see what you mean

Reply:
I'll think about it, will let you know if I have ideas. Quickly looking over this right now, it feels like toStrict is the best option so far as we do need to explicitly differentiate lazy vs required, and I'm not sure what other options there are.